### DISCUSSION TITLE
Cannot import name 'get_solver' from 'idaes.core.solvers

### DISCUSSION THREAD

**Discussion by anti-integral** [IDAES/idaes-pse#1006] [General]
<img width="1251" alt="Screen Shot 2022-11-01 at 6 13 42 PM" src="https://user-images.githubusercontent.com/68397796/199352020-8568ee3b-5170-4b6f-b7c6-cebf869ca5bb.png">

I don't know why this is happening after I imported idaes-pse.

**Comment by andrewlee94** [comment]
@osanan25 Could you please check the version of IDAES you are using? It is likely that you have an older version of IDAES (likely v1.13) as we have recently moved the `get_solver` function and updated the documentation and examples to reflect this. I suspect that you are using recent documentation with an older version of IDAES.
